cronjob.py

```diff
@@ -0,0 +1,81 @@
+
+from lib.modules.networkingmodule.networkingresponse import *
+import requests
+import json
+import numpy as np
+import os
+from dotenv import load_dotenv
+from pathlib import Path
+from aiohttp import ClientError
+from lib.modules.databasemodule.database import database
+db = database.instance()
+
+env_path = Path('.') / '.env'
+load_dotenv(dotenv_path=env_path)
+
+if __name__ == '__main__':
+    try:
+        header = {'Authorization': 'Bearer ' + os.getenv("SLACK_BOT_TOKEN")}
+        # Gets list of users in Networking channel
+        data = {'channel': 'C03QSC88Y4E'}
+        rawusers = requests.get('https://slack.com/api/conversations.members', headers=header, params=data).content
+        if not json.loads(rawusers.decode('utf-8'))['ok']:
+            print("Error: " + json.loads(rawusers.decode('utf-8'))['error'])
```

Comment:
recurring issue in this file is the way logging is done, logging should be done via logging.log not printing. Eventually this is because it will be integrated with CloudWatch and it will be very easy to debug